content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -33,6 +33,20 @@ spec:
 EOF
 {{< /text >}}
 
+To enable on a specific namespace, add a [`ProxyConfig` resource](/docs/reference/config/networking/proxy-config/):
```

Comment:
`ProxyConfig` can be mesh wide if its in `istio-system` FWIW

Reply:
Would it be better to recommend that approach for mesh wide enablement, as opposed to the istio-operator method?

Reply:
I am not sure we want to replace all references or any. I would either have each doc give a general description and link out to a broad doc on configuring Proxyconfig which has all options (annotation, mesh config, ProxyConfig) OR have tabs on the page giving each option?

Reply:
I think the tab approach can be done like:

```md
{{< tabset category-name="enable-dns-proxy-global" >}}
  {{< tab name="ProxyConfig" category-value="pc" >}}
    ...
  {{< /tab >}}
  {{< tab name="IstioOperator" category-value="iop" >}}
  ...
  {{< /tab >}}
{{< /tabset >}}
{{< tabset category-name="enable-dns-proxy-per-namespace" >}}
  {{< tab name="ProxyConfig" category-value="pc" >}}
    ...
  {{< /tab >}}
  {{< tab name="IstioOperator" category-value="iop" >}}
  ...
  {{< /tab >}}
{{< /tabset >}}
```

I'll work on a doc separate from this PR for explaining the ways to configure `ProxyConfig`, no need to worry about it here.